libs/sdk-ui-dashboard/src/presentation/filterBar/dateFilter/DateFilterConfigurationBody.tsx

```diff
@@ -0,0 +1,66 @@
+// (C) 2023 GoodData Corporation
+import React, { useCallback, useState } from "react";
```

Comment:
It is better if you can group the import libraries and import in the same project into separate groups.